libsolidity/inlineasm/AsmScopeFiller.cpp

```diff
@@ -106,6 +106,12 @@ bool ScopeFiller::operator()(Switch const& _switch)
 	return success;
 }
 
+bool ScopeFiller::operator()(ForLoop const& _forLoop)
+{
+	/// FIXME: implement
```

Comment:
sure it's useful, it will allow parallel work on the code generator and will avoid merge conflicts. Merge early, merge often!